### PR TITLE
[ConstitutiveLawsApp] Organizing constitutive laws files more efficiently

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
**📝 Description**
In this PR I am rearranging the CLs inside the ConstitutiveLawsApp in such a way that you can find them easily. The paths have been updated in the files and tests.

imagen

and inside..

imagen

### CODE REVIEW

**Review by loumalouomega — DISMISSED** [review]
I agree

**Comment by rubenzorrilla** [comment]
Very good! Thanks for the effort.

**Comment by AlejandroCornejo** [comment]
sems that I should do a clean compilation in Win to detect all the changes

**Review by loumalouomega — DISMISSED** [review]

**Review by loumalouomega — DISMISSED** [review]

File: applications/ConstitutiveLawsApplication/custom_constitutive/small_strains/damage/d_plus_d_minus_damage_masonry_3d.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@
 #include "custom_utilities/tangent_operator_calculator_utility.h"
 #include "custom_utilities/advanced_constitutive_law_utilities.h"
 #include "constitutive_laws_application_variables.h"
-#include "custom_constitutive/d_plus_d_minus_damage_masonry_3d.h"
```
Comment (loumalouomega) [inline applications/ConstitutiveLawsApplication/custom_constitutive/small_strains/damage/d_plus_d_minus_damage_masonry_3d.cpp]: image

**Review by loumalouomega — APPROVED** [review]

**Pull request merged**